service/core.go

```diff
@@ -138,16 +138,15 @@ func (c *Core) addAPIServerRoutes() {
 	// /auth/method intentionally requires no authentication
 	c.routerAPI.Handle("/auth/method", c.handler(handleAuthMethodGet)).Methods("GET")
 	c.authhandle("/events", handleEvents).Methods("GET")
-	c.authhandle("/pool", handlePoolList).Methods("GET")
+	c.authhandle("/ids", handleIDs).Methods("POST")
 	c.authhandle("/pool", handlePoolPost).Methods("POST")
 	c.authhandle("/pool/{pool}", handlePoolDelete).Methods("DELETE")
-	c.authhandle("/pool/{pool}", handlePoolGet).Methods("GET")
 	c.authhandle("/pool/{pool}", handlePoolPut).Methods("PUT")
-	c.authhandle("/pool/{pool}/add", handleAdd).Methods("POST")
-	c.authhandle("/pool/{pool}/delete", handleDelete).Methods("POST")
-	c.authhandle("/pool/{pool}/squash", handleSquash).Methods("POST")
-	c.authhandle("/pool/{pool}/staging", handleScanStaging).Methods("GET")
-	c.authhandle("/pool/{pool}/staging/{commit}", handleCommit).Methods("POST")
+	c.authhandle("/pool/{pool}/{branch}", handleBranchDelete).Methods("DELETE")
+	c.authhandle("/pool/{pool}", handleBranchPost).Methods("POST")
+	c.authhandle("/pool/{pool}/{branch}", handleBranchLoad).Methods("POST")
+	c.authhandle("/pool/{pool}/{branch}/merge", handleBranchMerge).Methods("POST")
+	c.authhandle("/pool/{pool}/{branch}/delete", handleDelete).Methods("POST")
```

Comment:
Usually rest has the object name before id. Maybe put the word `branch` after the poolId and before branchId?

Reply:
But the resource is named `/pool/poolID/branchID` then we are applying a verb to that resource, which is not exactly REST-y but I don't see how swapping it makes more REST-y sense, i.e., applying say the verb `delete` to the pool with an branchID under that...?

Reply:
@mccanne: I think the suggestion is to change `/pool/{pool}/{branch}` to `/pool/{pool}/branch/{branch}`. (I'm in favor.)

Reply:
What @nwt said